doc/source/user_guide/text.rst

```diff
@@ -15,8 +15,9 @@ There are two ways to store text data in pandas:
 
 1. ``object`` -dtype NumPy array.
 2. :class:`StringDtype` extension type.
+3. ``str`` -dtype (default from pandas 3.0).
 
-We recommend using :class:`StringDtype` to store text data.
+We recommend using the ``str`` dtype or :class:`StringDtype` to store text data.
```

Comment:
Or `string`. Maybe reword to something like

    We recommend __not__ using ``object`` dtype to store text data.